Interfaces/iTerm2 Help/index.html

```diff
@@ -1243,6 +1243,12 @@
 
 <p><br/><br/></p>
 
+<h4>Answerback String</h4>
```

Comment:
This file is auto-generated from `source/_includes/documentation-preferences.md` in the repo https://github.com/gnachman/iterm2-website. Please send a separate PR to update that file.